### ISSUE TITLE
Question about Inception score evaluation

### ISSUE THREAD

**Issue by GeondoPark** [snap-research/MoCoGAN-HD#11]
Hello
Thank you for your great work! I read the paper carefully.

I wonder how to calculate the inception score of UCF-101 in detail.
I read that you follow the tgan paper for evaluating the inception scores and use the C3D network for getting the predictions. 

Which weights did you use for the C3D network? Did you train from the initial?
If not, could you let me know the weight of the C3D network and w how to use the C3D net?

In detail, in this paper, the size of the generated video(ucf101) is (224, 224). But pre-trained network of C3D in this link (https://github.com/rezoo/tgan2/releases/download/v1.0/conv3d_deepnetA_ucf.npz) was not trained with config (224, 224). How did you resize and normalize the frames..? 

I would be very grateful if you could reply.
Thanks.

**Comment by alanspike** [comment]
Hey! We didn't finetune C3D network. We used the released weights. We resize the frames generated by our method using `cv2` in Python with interpolation as `INTER_CUBIC`.
